org.torproject.torbrowser-launcher.yaml

```diff
@@ -1,15 +1,15 @@
 app-id: org.torproject.torbrowser-launcher
 runtime: org.kde.Platform
-runtime-version: 5.15-23.08
+runtime-version: 6.8
 sdk: org.kde.Sdk
 base: com.riverbankcomputing.PyQt.BaseApp
-base-version: 5.15-23.08
+base-version: 6.7
```

Comment:
FYI PyQt 6.8 should be released in a week or two (after the Qt 6.8.1 release). Its Flatpak branch will be added afterwards).